src/lib.rs

```diff
@@ -213,14 +209,11 @@ impl DocData {
         for def in defs.iter() {
```

Comment:
yeah i was certainly cargo-culting this code; getting rid of eventual clones would be great!